internal/venice-common/src/main/java/com/linkedin/venice/meta/VeniceUserStoreType.java

```diff
@@ -17,11 +17,13 @@ public enum VeniceUserStoreType {
   /**
    * The data in these stores are from full pushes and incremental pushes.
    */
+  @Deprecated
```

Comment:
Looks like this enum config has been used in other places, have you check if this will impact other code path? Like batch enabling NR/AA for a cluster?

Reply:
No, it won't have any impact on enabling NR/AA.